src/common/engines/gemini.ts

```
@@ -4,174 +4,174 @@ import { AbstractEngine } from './abstract-engine'
+        hasError = true
+        if (err instanceof Error) {
+          req.onError(err.message)
+          return
+        }
+        if (typeof err === 'string') {
+          req.onError(err)
+          return
+        }
+        if (typeof err === 'object') {
+          const item = err[0]
+          if (item && item.error && item.error.message) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)